modelling/feat_engineer/feature_engineering.py

```diff
@@ -10,10 +11,54 @@
 from sklearn.feature_selection import SelectKBest, f_classif
 import pywt
 from ..lamonaca_and_nemcova.utils import spo2_estimation, get_peak_height_slope
+from tqdm import tqdm
+
+
+def _eval_feat_engineering(all_features):
+    # Little helper function I wrote for some analysis comparing HR and SpO2 as a target
+
+    # https://github.com/gianlucatruda/Spo2_evaluation/issues/4
+    from sklearn.metrics import mean_squared_error as mse
+    from sklearn.linear_model import LassoLars, LinearRegression
```

Comment:
Is there a reason behind choosing Lars method exactly?

Reply:
Lasso is one of the best linear models in terms of regularisation. It can penalise features all the way to zero, so it effectively achieves the same thing as feature selection. I could use something other than LARS for the implementation, but [from what I've read](https://scikit-learn.org/stable/modules/linear_model.html#least-angle-regression) it's a good way to do things.